### ISSUE TITLE
How to use WinSCP

### ISSUE THREAD

**Issue by plrnr** [Eugeny/terminus#3138]
Version: 1.0.120
Platform: win32 10.0.18363
Plugins: scrollbar

I entered WinSCP full path in SSH configuration. How do I use WinSCP? Right click on the tab does not show anything about WinSCP.

**Comment by Lireoy** [comment]
WinSCP is a separate program. https://winscp.net/eng/index.php
WinSCP has no accociation with Terminus.

**Comment by plrnr** [comment]
thanks but that does not sound right. There has to be a purpose for the WinSCP path found in SSH. See screenshot.

![image](https://user-images.githubusercontent.com/32370793/101810753-0070d280-3ace-11eb-9387-14c8d0bf71f7.png)

**Comment by bernardyip** [comment]
Right-click your SSH session and click "Launch WinSCP"
![image](https://user-images.githubusercontent.com/1573143/101970302-96d6fc80-3c64-11eb-812f-be3aa0dbc533.png)

**Comment by plrnr** [comment]
Thanks @bernardyip . Found out that it is visible in a right click on terminal but not on rt click on the tab itself at the top. In addition, I had rt. click set to "paste" in settings, so I had to change that to expose WinSCP option.
WinSCP really should be available from rt clicking on the tab.

![image](https://user-images.githubusercontent.com/32370793/102808307-fe7ffc80-4374-11eb-84c1-f94f4d06fe39.png)
